### DISCUSSION TITLE
Questions about enable my account Sponsors feature

### DISCUSSION THREAD

**Discussion by chyroc** [community/community#39653] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hi, everyone, github administrators, hello.

I am chyroc, a developer from China and a user of github.

I applied to activate the Sponsors feature of github a long time ago, but the status of the apply has been pending. I have waited for a long time, maybe 1-2 years.

May I ask if the Sponsors feature is no longer available to the public. If not, I very much hope that the github administrators can review my Sponsors apply (if I am qualified), thank you very much.

I'm sorry if the inquiries about this issue should be posted in this place, please close my question, thank you.

**Comment by david-engelmann** [comment]
The sponsors feature is still available to the public. It's very possible its not a feature rolled out in your country yet. Check out the supported countries in the screenshot below. [You can go to this sight to get all the info on Github Sponsors](https://github.com/sponsors).

![image](https://user-images.githubusercontent.com/81196676/203097716-6b321870-e7f9-425e-8280-c12f4e8c5ec0.png)

**Reply by chyroc** [reply]
i see region [HongKong] is supported, and i select it as my Bank account country or region.
